### PR TITLE
Fixing versions

### PR DESCRIPTION
**Pull request by vxbinaca**
- Updated tubeup to 0.0.20
- Switched to youtube-dlc

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -12,6 +12,6 @@ schema==0.7.2
 six==1.15.0
 total-ordering==0.1.0
 tqdm==4.50.2
-tubeup==0.0.19
+tubeup==0.0.20
 urllib3==1.25.11
-youtube-dl==2020.9.20
+youtube-dlc==2020.9.10.26
```
Comment (etnguyen03) [inline requirements.txt:17]:
```suggestion
youtube-dlc==2020.10.26
```

**Review by etnguyen03 — CHANGES_REQUESTED** [review]
I think this is why it's failing:

File: requirements.txt (hunk 1 above)
Comment (vxbinaca) [reply requirements.txt]: yeah sorry was early in the morning when I did that.

**Comment by vxbinaca** [comment]
> I think this is why it's failing:

She's good to go

**Comment by etnguyen03** [comment]
Merged in f667ea03d006d97887e80412c5211d3d0dfa333b.

Closing

**Pull request closed without merging**